CODEOWNERS

```diff
@@ -202,20 +211,43 @@
 /samples/cellular/modem_trace_flash/      @eivindj-nordic
 /samples/cellular/slm_shell/              @nrfconnect/ncs-iot-oulu
 /samples/cellular/sms/                    @trantanen @tokangas
-/samples/openthread/                      @nrfconnect/ncs-co-networking @nrfconnect/ncs-thread
+/samples/common/                          @nrfconnect/ncs-si-muffin @nrfconnect/ncs-si-bluebagel
+/samples/crypto/                          @nrfconnect/ncs-aegir
+/samples/debug/memfault/                  @nrfconnect/ncs-cia
+/samples/debug/ppi_trace/                 @nordic-krch
+/samples/dect/dect_phy/			  @jhirsi
```

Comment:
`dect_shell` is @jhirsi.
`hello_dect` should be @nrfconnect/ncs-modem or @eivindj-nordic. Can you comment @eivindj-nordic?

Reply:
I'll add both